test/core/util/avl_fuzzer.cc

```diff
@@ -19,8 +19,8 @@
 #include <map>
 #include <utility>
 
+#include "fuzztest/fuzztest.h"
 #include "src/core/util/avl.h"
-#include "src/libfuzzer/libfuzzer_macro.h"
 #include "test/core/util/avl_fuzzer.pb.h"
 
 bool squelch = true;
```

Comment:
Are `squelch` and `leak_check` still needed?

Reply:
just saw this.. not really.. i'll cleanup as a separate pass